pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -136,14 +136,15 @@ def connection_select(context, name, **options):
     """
     Select a WBEM connection definition as current or default.
 
-    Selects a connection from the persistently stored named connections if NAME
-    exists in the store to be the current connection. If the NAME argument does
-    not exist, a list of connections from the connections definition file is
-    presented with a prompt for the user to select a connection.
+    Selects the connection named NAME from the persistently stored named
+    connectionsto be the current connection if NAME exists in the store . If
```

Comment:
Remove "if NAME exists in the store". If you want to keep it, change it to "if it exists in the connections file", but I think we should describe the normal case here. The exceptional case is described in the next sentence.

Reply:
DONE